frontend/src/App.svelte

```diff
@@ -211,18 +203,12 @@ Answer by running riddle.answer('your answer here') from the console.`);
         scrollY="{hexScrollY}"
         displayMinimap="{currentResource && !useAssemblyView && !useTextView}"
       >
-        {#if useAssemblyView}
-          <AssemblyView />
-        {:else if useTextView}
-          <TextView />
-        {:else}
-          <HexView
-            dataLenPromise="{dataLenPromise}"
-            resources="{resources}"
-            scrollY="{hexScrollY}"
-            bind:resourceNodeDataMap="{resourceNodeDataMap}"
-          />
-        {/if}
+        <DataView
```

Comment:
Nitpicky, but I feel like maybe `DataView` is a misleading name since "data" in OFRAK usually refers to raw bytes. I suppose the text and assembly views are just another way of decoding that same data, but the decompilation view is further disconnected and I would prefer a more unique name less likely to mislead. Something like `ResourceBodyView` or `ContentsView`, maybe.

Reply:
ContentsView is good, I didn't think about this name much